lemur/certificates/views.py

```diff
@@ -1133,6 +1138,7 @@ def __init__(self):
         super(CertificateUpdateOwner, self).__init__()
 
     @validate_schema(certificate_edit_input_schema, certificate_output_schema)
+    @operator_permission.require(http_exception=403)
```

Comment:
same comment as update endpoint w.r.t. owners being able to update metadata for their certs